devConfig.yaml

```diff
@@ -83,4 +83,16 @@ core_config_version: 0
 disable_telemetry: true
 
 # (OPTIONAL | Default: ""). Used to prepend a base path to all APIs when querying the core.
-# base_path:
\ No newline at end of file
+# base_path:
+
+# (OPTIONAL | Default: "ARGON2"). The password hashing algorithm to use. Values are "ARGON2" | "BCRYPT"
```

Comment:
Is the new default intentional? It's not mentioned in the changelog.

Reply:
Yes. Changelog is now updated.